src/test/java/org/opentripplanner/graph_builder/module/osm/NorwayWayPropertySetSourceTest.java

```diff
@@ -0,0 +1,37 @@
+package org.opentripplanner.graph_builder.module.osm;
+
+import org.junit.Test;
+import org.opentripplanner.openstreetmap.model.OSMWithTags;
+import org.opentripplanner.routing.edgetype.StreetTraversalPermission;
+
+import static org.junit.Assert.assertEquals;
+
+public class NorwayWayPropertySetSourceTest {
+  static WayPropertySet wps = new WayPropertySet();
+
+  static {
+    var source = new NorwayWayPropertySetSource();
+    source.populateProperties(wps);
+  }
+
+  @Test
+  public void testMtbScaleNone() {
+    // https://www.openstreetmap.org/way/302610220
+    var way = new OSMWithTags();
+    way.addTag("highway", "path");
+    way.addTag("mtb:scale", "3");
+
+    assertEquals(
+        wps.getDataForWay(way).getPermission(), StreetTraversalPermission.NONE);
+  }
+
+  @Test
+  public void testMtbScalePedestrian() {
```

Comment:
I like this test but after having looked around the the OSM data in Norway, I found that the `mtb:scale` tag is also used in combination with `highway=track`.

Examples:

https://www.openstreetmap.org/way/514943207
https://www.openstreetmap.org/way/370993765

I think it'd be worthwhile to add a test for this, too, to prevent someone breaking it by accident.